tests/acceptance/pageObjects/sharedWithMePage.js

```diff
@@ -13,6 +13,18 @@ module.exports = {
       return navigationHelper.navigateAndWaitTillLoaded(
         this.url(), this.page.FilesPageElement.filesList().elements.filesListProgressBar
       )
+    },
+    assertPendingStatusIsPresent: function (filename) {
+      return this.waitForElementVisible({
+        locateStrategy: this.elements.pendingStatus.locateStrategy,
+        selector: this.api.page.FilesPageElement.filesList().getFileRowSelectorByFileName(filename) + this.elements.pendingStatus.selector
+      })
+    }
+  },
+  elements: {
+    pendingStatus: {
```

Comment:
```suggestion
    pendingStatusFileRow: {
```
I think using it this way makes it clear that this xpath works from FileRowXpath not the root. What do you think?